sa_mobile/SASCM.ini

```diff
@@ -1968,70 +1968,90 @@ DATE=2020-11-11
-0a73=4,set_widget_value3 %1d% %2d% %3d% %4d%
-0a74=1,  is_checkpoint_resuming %1d%
-0a75=2,set_active_menu_item_car_mods %1d% %2d%
+0a70=4,display_text_clamped_position %1d% %2d% GXT %3g% scale %4d% ;text_draw  
+0a71=5,display_text_clamped_position %1d% %2d% GXT %3g% with_number %4d% scale %5d% ;text_draw
+0A72=0,auto_save_checkpoint
+0a73=4,set_widget %1d% value3 %2d% %3d% %4d% ; float
+0a74=1,  is_checkpoint_resuming %1d% ; IsGameResuming //bool
+0a75=2,  set_active_square_color_panel %1d% find_car_colour %2d%
 0a76=1,  hid_implements %1d%
 0a77=1,checkpoint_save_oddjob %1d%
-0a78=2,print_help_conditional %1g% %2d%
-0a79=2,print_help_conditional_touch %1g% %2d%
-0a7a=2,print_help_conditional_hid %1g% %2d%
-0a7b=2,print_help_conditional_touch_classic %1g% %2d%
-0a7c=2,print_help_conditional_touch_adapted %1g% %2d%
-0a7d=2,print_help_conditional_hid_joypad %1g% %2d%
-0a7e=2,print_help_conditional_hid_keyboard %1g% %2d%
-0a7f=2,print_help_conditional_touch_analog %1g% %2d%
-0a80=2,print_help_conditional_unknown %1g% %2d%
-0a81=2,set_widget_info2 %1g% %2d%
-0a82=2,set_widget_info2 %1g% %2d%
-0a83=2,set_widget_info2 %1g% %2d%
-0a84=2,set_widget_info2 %1g% %2d%
-0a85=2,set_widget_info2 %1g% %2d%
-0a86=2,set_widget_info2 %1g% %2d%
-0a87=2,set_widget_info2 %1g% %2d%
-0a88=2,set_widget_info2 %1g% %2d%
-0a89=3,set_widget_value %1d% %2d% %3d%
-0a8a=8,set_widget_info %1d% %2d% %3d% %4d% %5d% %6d% %7d% %8g%
-0a8b=7,set_widget_info %1d% %2d% %3d% %4d% %5d% %6g% %7g%
-0a8c=0,load_all_streaming_models
+0A78=2,print_help_forever_conditional_touch %1g% hid_type %2d%
+0A79=2,print_help_forever_conditional_hid %1g% hid_type %2d%
+0A7A=2,print_help_forever_conditional_touch_classic %1g% hid_type %2d%
+0A7B=2,print_help_forever_conditional_touch_adapted %1g% hid_type %2d%
+0A7C=2,print_help_forever_conditional_hid_joypad %1g% hid_type %2d%
+0A7D=2,print_help_forever_conditional_hid_keyboard %1g% hid_type %2d%
+0A7E=2,print_help_forever_conditional_touch_analog %1g% hid_type %2d%
+0A7F=2,print_help_forever_conditional_touch_digital %1g% hid_type %2d%
+0A80=2,print_help_forever_conditional_touch_flick %1g% hid_type %2d%
+0A81=2,print_help_conditional %1g% hid_type %2d%
+0A82=2,print_help_conditional_touch %1g% hid_type %2d%
+0A83=2,print_help_conditional_hid %1g% hid_type %2d%
+0A84=2,print_help_conditional_touch_classic %1g% hid_type %2d%
+0A85=2,print_help_conditional_touch_adapted %1g% hid_type %2d%
+0A86=2,print_help_conditional_hid_joypad %1g% hid_type %2d%
+0A87=2,print_help_conditional_hid_keyboard %1g% hid_type %2d%
+0A88=2,print_help_conditional_touch_analog %1g% hid_type %2d%
+0a89=3,set_widget %1d% value2 %2d% %3d% ; float 
+0a8a=8,set_widget %1d% info %2d% %3d% %4d% %5d% %6d% %7d% %8g%
+0a8b=7,set_widget %1d% info2 %2d% %3d% %4d% %5d% %6g% %7g%
+0a8c=0,load_all_requested_models ; No time to pause. load faster than 038B
 0a8d=1,  is_hid_released %1d%
-0A8E=1,award_achievement %1d%
-0A8F=0,
-0A90=1,  %1d%
\ No newline at end of file
+0A8E=1,NOP %1d%
```

Comment:
Are you sure this is a no-operation? I remember there was a discussion somewhere on GTAForums about this opcode and somehow it was figured this grants new achievements to the player. I haven't tested it myself though

Reply:
https://gtaforums.com/topic/929701-new-opcodes-in-the-remastered-scm-of-san-andreas/?do=findComment&comment=1070954748

Reply:
@XMDS ☝️